tools/docker-compose/README.md

```diff
@@ -116,7 +166,7 @@ awx_1        |   Applying auth.0001_initial... OK
 ...
 ```
 
-##### Clean and Build UI
+##### Clean and build the UI
```

Comment:
3 out of 5 of these header changes are to add title-casing.  Shouldn't this be the same?